<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PurgeHistoryWidget</class>
 <widget class="QWidget" name="PurgeHistoryWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Purge History</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,100">
     <item>
      <widget class="QLabel" name="label_cdnInstanceId_text">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="contextMenuPolicy">
        <enum>Qt::DefaultContextMenu</enum>
       </property>
       <property name="text">
        <string>CDN Instance ID:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_cdnInstanceId_value">
       <property name="text">
        <string>123456</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,100">
     <item>
      <widget class="QLabel" name="label_queryTime_text">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>조회 일시:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_queryTime_value">
       <property name="text">
        <string>20XX-XX-XX HH:MM:SS</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="label_description">
     <property name="font">
      <font>
       <weight>50</weight>
       <italic>true</italic>
       <bold>false</bold>
       <underline>false</underline>
       <kerning>true</kerning>
      </font>
     </property>
     <property name="text">
      <string>*최근 5회동안의 Purge 로그를 확인할 수 있습니다.</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTableView" name="tableView_historyList"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
